src/app/frontend/common/components/workloadstatus/style.scss

```diff
@@ -15,7 +15,9 @@
 @import '../../../variables';
 
 .kd-graph-container {
-  flex: auto;
```

Comment:
Why drop flex?

Reply:
It's not ready yet as you can see in the title. I still have to fix the styling.